examples/using-preact/next.config.js

```diff
@@ -14,6 +14,11 @@ module.exports = withPrefresh({
       }
     }
 
+    if (isServer) {
+      // mark `preact` stuffs as external for server bundle to prevent duplicate copies of preact
+      config.externals.push(/^preact/);
```

Comment:
Probably best to use the regex from the chunk configuration above - that was this doesn't end up marking something like `preact-markup` as external:

```suggestion
      config.externals.push(/^(preact|preact-render-to-string|preact-context-provider)([\\/]|$)/);
```